src/components/Dashboard/UsersGraph/UsersGraph.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts';
+import store from '../../../store';
+import _ from 'lodash';
+import useInterval from '@use-it/interval';
+import moment from 'moment';
+import { TileContainer, GraphTitle } from './style';
+
+interface dataPoint {
+  hour: string;
+  Users: number;
+}
+
+const getGraphData = () => {
+  const initialData = [
```

Comment:
Can this array be initialized with a for loop to make the code more compact? Just keep appending to the array.

Reply:
Temp solution for now. Since this is an initial state, perhaps we can put this in redux instead in the future?